.controlplane/controlplane.yml

```diff
@@ -0,0 +1,30 @@
+# Configuration for "Heroku to CPLN playbook" custom scripts
+aliases:
+  common: &common
+    # Change this to your org name for staging. Production apps will use a different org
+    # for security.
+    cpln_org: shakacode-open-source-examples
+    # Change `shakacode-staging` to your-org-name-for-staging
+    # Example apps use only location. CPLN offers the ability to use multiple locations.
+    default_location: aws-us-east-2
+    # Configure the workload name used as a template for one-off scripts, like a Heroku one-off dyno.
+    one_off_workload: rails
+    # Like the entries in the Heroku Procfile that get deployed when the application code changes
+    # and the application image updates.
+    app_workloads:
+      - rails
+    # Like Heroku add-ons
+    # additional_workloads:
+    #   - postgres
+
+apps:
+  react-rails-example-app:
+    <<: *common
+  tutorial-app:
+    <<: *common
+  # QA Apps are like Heroku review apps, but the use `prefix` so you can run a commmand like
+  # this to create a QA app for the tutorial app.
+  # `cpl setup gvc postgres redis rails -a qa-tutorial-app-pr-1234`
+  qa-tutorial-app:
+    <<: *common
+    prefix: true
```

Comment:
I kept these lines similar to RWRT project. But I don't think we need them. Should I remove them?